app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java

```diff
@@ -622,14 +629,14 @@ else if (carta.getValorTruco(s.manilha) == 14)
 		// abaixo
 		Carta tres_testedemanilha = new Carta('3', 3);
 		Carta dois_testedemanilha = new Carta('2', 3);
-		if (qcarta == DOIS
+		if (qcarta == Qualificacao.DOIS.value()
 				&& tres_testedemanilha.getValorTruco(s.manilha) == 14)
-			qcarta = TRES;
-		if (qcarta == AS && tres_testedemanilha.getValorTruco(s.manilha) == 14)
-			qcarta = DOIS;
+			qcarta = Qualificacao.TRES.value();
+		if (qcarta == Qualificacao.AS.value() && tres_testedemanilha.getValorTruco(s.manilha) == 14)
+			qcarta = Qualificacao.DOIS.value();
 		// caso a manilha seja um 2, o A passa a ter valor de 2!
-		if (qcarta == AS && dois_testedemanilha.getValorTruco(s.manilha) == 14)
-			qcarta = DOIS;
+		if (qcarta == Qualificacao.AS.value() && dois_testedemanilha.getValorTruco(s.manilha) == 14)
+			qcarta = Qualificacao.DOIS.value();
 
 		return qcarta;
```

Comment:
What do you think about declaring `qcarta` as a `Qualificacao` and just converting to integer in the end?

Alternative: changing the return type of this method to `Qualificacao` (I'm not sure *all* uses of it are compatible with that, but the ones below look like so)